src/timeunit.ts

```diff
@@ -294,7 +294,8 @@ export function template(timeUnit: TimeUnit, field: string, shortTimeLabels: boo
    const template = '{{' + field + ' | time:\'' + out.join(' ') + '\'}}';
 
    // FIXME: Remove this RegExp Hack!!!
-   return template.replace(new RegExp('{{' + field + ' \\| time:\'\'}}', 'g'), '');
+   const escapedField = field.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
```

Comment:
Also, I'm having some difficulty following the timeunit method. I sort of see that the way it's being built up sometimes adds these empty templates... but wasn't really sure why it was initially written this way. I can take a look soon :)